### DISCUSSION TITLE
Credentials CCU2

### DISCUSSION THREAD

**Discussion by hattmann** [SukramJ/aiohomematic#378] [Q&A] [answered]
Hello, a small question about Credentials for the Homematic CCU:

On my CCU2 is authentication currently disabled (for openHAB). 

In your implementation i must provide Username/Passwort for the CCU. 
Is there a config file where i can just change the credentials i have entered via the webgui to null values?
I need this for testing beforce making the change to home assistant. 
The XML RPC Log reports wrong credentials if i enter sample data like admin/admin.

**Comment by SukramJ** [comment, marked as answer]
Due to the use of JsonRPC authentication is required for this integration.

**Reply by hattmann** [reply]
Ok, thank you!
